network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -39,7 +39,7 @@
                   </div>
                 </div>
               </div>
-              <a class="btn btn-secondary" href="/proposals">Submit proposal</a>
+              <a class="btn btn-secondary" href="{{ menu_root.specific.cta_button_destination }}">{{ menu_root.specific.cta_button_label }}</a>
```

Comment:
Can be a followup - can we make external link to open on a new tab?

Reply:
That feels like something we should be looking at when we actually change the button to link to an external site, because it's entirely possible we'll never do and always funnel to our own pages first.

Reply:
/cc @alanmoo @gideonthomas

Reply:
yeah I wouldn't worry about it